next.config.js

```diff
@@ -59,7 +61,6 @@ module.exports = (phase, { defaultConfig }) => {
 
       return config
     },
-    i18n,
```

Comment:
Important change. This is not supported in the app router, removing now and building the pages using `getStaticPaths`